guides/common/assembly_managing-activation-keys.adoc

```diff
@@ -1,47 +1,4 @@
-[id="Managing_Activation_Keys_{context}"]
-= Managing Activation Keys
```

Comment:
The title and perhaps an intro paragraph should stay in the assembly.
Mark -- see my comment for `assembly_synchronizing-content-between-foreman.adoc`